src/main/resources/application.properties

```diff
@@ -1,3 +1,6 @@
-spring.data.mongodb.database=springangular
-spring.data.mongodb.host=localhost
-spring.data.mongodb.port=27017
\ No newline at end of file
+spring.data.mongodb.host=54.93.39.245
+spring.data.mongodb.port=27017
+spring.data.mongodb.authentication-database=assets
+spring.data.mongodb.username=gd2
+spring.data.mongodb.password=javaee
```

Comment:
W sumie takich informacji o tym jak dostać się do bazy nie powinniśmy na gita wrzucać :thinking:  ale nie wiem jak w sumie jak to inaczej można trzymać więc idk.